unlock-app/src/components/content/MembersContent.tsx

```diff
@@ -74,6 +76,12 @@ export const MembersContent = ({ query }: MembersContentProps) => {
     page = parseInt(query.page)
   }
 
+  const hasLocks = lockAddresses?.length > 0
+
+  const onLockChange = (lock: Lock) => {
+    window.location.href = `/members?locks=${lock.address}`
```

Comment:
I don't think it's the right approach because it will force a reload if the page

Reply:
change with `useRouter ` hook